src/networking/client/plugin.rs

```diff
@@ -0,0 +1,56 @@
+use std::net::SocketAddr;
+
+use bevy::prelude::{AppBuilder, Plugin, IntoQuerySystem};
+use quinn::{crypto::rustls::TlsSession, generic::Connecting};
+use tokio::sync::mpsc::unbounded_channel;
+use url::Url;
+
+use crate::networking::{events::ReceiveEvent, events::SendEvent, systems::NetworkConnections, systems::SessionEventListenerState, systems::handle_connection, systems::receive_net_events, systems::send_net_events};
```

Comment:
This should probably be unfolded... I think rust-analyzer can even do that for you.

Reply:
I merged everything to `use crate::networking::{events::{ReceiveEvent, SendEvent}, systems::{NetworkConnections, SessionEventListenerState, handle_connection, receive_net_events, send_net_events}};`